src/utils/themes-manager.js

```diff
@@ -0,0 +1,79 @@
+//@flow
+
+import {hexToHsl} from './color-format-convertors';
+import {style} from '../index';
+
+const cssVarNames: ThemesManagerConfig = {
+  colors: {
+    primary: '--playkit-primary-hsl-hue',
+    secondary: '--playkit-secondary-hsl-hue',
+    success: '--playkit-success-hsl-hue',
+    danger: '--playkit-danger-hsl-hue',
+    warning: '--playkit-warning-hsl-hue',
+    live: '--playkit-live-color',
+    playerBackground: '--playkit-player-background-color'
+  }
+};
+
+const dynamicColoredIconsSvgUrlVars = [
+  '--playkit-icon-check-active-url',
+  '--playkit-icon-data-url',
+  '--playkit-icon-chromecast-url',
+  '--playkit-icon-quality-HD-active-url',
+  '--playkit-icon-quality-4K-active-url',
+  '--playkit-icon-quality-8K-active-url'
+];
+
+// eslint-disable-next-line require-jsdoc
```

Comment:
you can disable it for the entire file at once.
do you want to add this in a separate task ?

Reply:
fixed